t/update.t

```diff
@@ -0,0 +1,41 @@
+use strict;
+use Test::More;
+use File::Temp qw/tempfile/;
+use FindBin qw($Bin);
+use File::stat;
+use Path::Tiny;
+
+my $sanction_file;
+
+BEGIN {
+    (my $fh, $sanction_file) = tempfile();
+    print $fh "ABCD\n";
+    close($fh);
+    $ENV{SANCTION_FILE} = $sanction_file;
+}
+use Data::Validate::Sanctions qw/is_sanctioned/;
+
+is($Data::Validate::Sanctions::sanction_file, $sanction_file, "sanction file is correct");
+
+ok(is_sanctioned('ABCD'),  "correct file content");
+ok(!is_sanctioned('AAAA'), "correct file content");
+
+#sleep 1 to make the mtime greater than old mtime
+my $last_mtime = stat($sanction_file)->mtime;
+sleep 1;
+my $script = "$Bin/../script/update_sanctions_csv";
+my $lib    = "$Bin/../lib";
+is(system($^X, "-I$lib", $script, $sanction_file), 0, "download file successfully");
```

Comment:
This could cause some spurious failures on cpantesters - they don't all allow network access. I think just make this an author test instead, don't want our useragent to be blocked by the sanctions website just because lots of cpantesters systems start downloading at the same time!